RecoEgamma/EgammaElectronProducers/python/lowPtGsfElectronID_cff.py

```diff
@@ -11,10 +11,20 @@
     )
 
 from Configuration.ProcessModifiers.run2_miniAOD_UL_cff import run2_miniAOD_UL
-run2_miniAOD_UL.toModify(
+from Configuration.Eras.Modifier_run2_miniAOD_devel_cff import run2_miniAOD_devel
+from Configuration.Eras.Modifier_bParking_cff import bParking
+( run2_miniAOD_UL | bParking ).toModify(
     lowPtGsfElectronID,
     rho = "fixedGridRhoFastjetAll",
     ModelWeights = ["RecoEgamma/ElectronIdentification/data/LowPtElectrons/LowPtElectrons_ID_2020Sept15.root"],
     ModelThresholds = [-99.],
     Version = "V1",
 )
+run2_miniAOD_devel.toModify(
+    lowPtGsfElectronID,
+    ModelWeights = ["RecoEgamma/ElectronIdentification/data/LowPtElectrons/LowPtElectrons_ID_2020Nov28.root"],
+)
+bParking.toModify(
```

Comment:
```suggestion
(bParking & run2_miniAOD_UL).toModify(
```
IIUC, this will properly keep the reco unchanged to resolve the difference in the reco output mentioned in 
https://github.com/cms-sw/cmssw/pull/33589#issuecomment-855139509

Reply:
Yes, good catch.